src/jitterentropy-noise.c

```diff
@@ -77,7 +77,7 @@ static uint64_t jent_loop_shuffle(struct rand_data *ec,
 	 * We add a lower boundary value to ensure we have a minimum
 	 * RNG loop count.
 	 */
-	return (shuffle + (1U<<min));
+	return (shuffle + (1UL<<min));
```

Comment:
Could you change `1UL` to `UINT64_C(1)` here? I think that is the correct value. That should resolve to the correct type across platforms.